tests/N3Tests/manifest-parser.ttl

```diff
@@ -1050,15 +1050,15 @@
 :isImpliedBy_graphs
         a              test:TestN3Eval ;
         mf:action      <isImpliedBy/isImpliedBy_graphs.n3> ;
-        mf:result      <isImpliedBy/isImpliedBy_graphs.nq> ;
+        mf:result      <isImpliedBy/isImpliedBy_graphs_result.n3> ;
         mf:name        "isImpliedBy Eval";
         rdfs:comment   "<= should be parsed as log:isImpliedBy in a complex use case";
```

Comment:
unsure what is meant by complex use case?

Reply:
Not sure either - resolved in https://github.com/w3c/N3/pull/202/commits/59f2ae74df01baa1ee2c3bad901ce578c644c252.

I don't have permissions to merge on this repo so will leave that to you.